File: java/get-localized-content/getting_localized_content_url_slug.java
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("KONTENT_AI_ENVIRONMENT_ID");

// Tip: Create strongly typed models according to https://kontent.ai/learn/net-strong-types
// Registers the model class for articles
client.registerType(Article.class);

// Filters all articles to find the Spanish variant by its URL slug
CompletionStage<List<Article>> items = client.getItems(
    Article.class,
    DeliveryParameterBuilder.params()
        .language("es-ES")
        .filterEquals("system.type", "article")
        .filterEquals("elements.url_pattern", "acerda-de-nosotros")
        .build();
);

File: php/get-localized-content/getting_localized_content_url_slug.php
<?php
// Tip: Find more about PHP SDKs at https://kontent.ai/learn/php
// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kontent\Ai\Delivery\DeliveryClient;

$client = new DeliveryClient('8d20758c-d74c-4f59-ae04-ee928c0816b7');

// Filters all articles to find the Spanish variant by its URL slug
$items = $client->getItems((new QueryParams())
            ->language('es-ES')
            ->equals('system.type', 'article')
            ->equals('elements.url_pattern', 'acerda-de-nosotros'));
?>